metrics/SystemMetrics.md

```diff
@@ -0,0 +1,35 @@
+# System Metrics
+This document serves to document the "look and feel" of the OpenCensus API for auto-collection of system metrics such as processes, memory & CPU usage, disks, etc.
+
+## Supported platforms
+TODO: Add list of supported platforms.
+
+## Common metrics collection
+These are some default metrics supported by OpenCensus system metrics package/contrib. This is to ensure consistency across language implementations, as far as practical.
+
+| Metric Name                               | Unit  | Type                       | Description                                                                                                | Labels                 |
+|-------------------------------------------|-------|----------------------------|-----------------------------------------------------------------------------------------------|-----------------|
+| system/cpu_seconds/total                  | s     | Int64 Cumulative           | Total kernel/system user CPU in seconds.                                                          | Hostname, Mode          |
+| system/processes/created                  | 1     | Int64 Cumulative           | Total number of times a process was created.                                                      | Hostname                |
+| system/processes/running                  | 1     | Int64 Gauge                | Total number of running processes.                                                                | Hostname                |
```

Comment:
add blocked process
system/processes/blocked, 1, Int64 Gauge, Number of processes blocked.

Reply:
Done, thanks.